Test 'Parser.write should use special characters as defined in the UNA header' fail

Hi,

On development branch with node 7.2.0, tests suite fail on 'Parser.write should use special characters as defined in the UNA header'.

```bash
> edifact@1.2.7 test /home/christophe/Dev/tdecaluwe/node-edifact
> istanbul cover -i "[^ies]*.js" --include-all-sources jasmine

Started
............F................................

Failures:
1) Parser.write should use special characters as defined in the UNA header
  Message:
    Expected function not to throw, but it threw Error: Cannot close an incomplete message.
  Stack:
    Error: Expected function not to throw, but it threw Error: Cannot close an incomplete message.
        at Object.<anonymous> (/home/christophe/Dev/tdecaluwe/node-edifact/spec/ParserSpec.js:20:46)

45 specs, 1 failure
Finished in 0.035 seconds

=============================================================================
Writing coverage object [/home/christophe/Dev/tdecaluwe/node-edifact/coverage/coverage.json]
Writing coverage reports at [/home/christophe/Dev/tdecaluwe/node-edifact/coverage]
=============================================================================

=============================== Coverage summary ===============================
Statements   : 83.02% ( 357/430 )
Branches     : 81.21% ( 134/165 )
Functions    : 72.6% ( 53/73 )
Lines        : 83.02% ( 357/430 )
================================================================================
npm ERR! Test failed.  See above for more details.
```

As I was looking why, I suppose this is because Tokenizer is instantiated before settings configuration from UNA header and take default Configuration settings.

Do you have any idea how to solve this ?

--- a/parser.js
+++ b/parser.js
@@ -94,6 +94,8 @@ Parser.prototype.una = function (chunk) {
     this._configuration.DM = chunk.charCodeAt(5);
     this._configuration.RC = chunk.charCodeAt(6);
     this._configuration.ST = chunk.charCodeAt(8);
+
+    this._tokenizer = new Tokenizer(this._configuration);
     return true;
   } else {
     return false;

